docs/tutorials/mastering.md

```diff
@@ -897,52 +1102,118 @@ Due to the newgrp command you will need to log out twice as it started a new she
 
 ## 4. Deploy SD-Core Control Plane
 
-The following steps build on the Juju controller which was bootstrapped and knows how to manage the SD-Core Control Plane Kubernetes cluster.
+The following steps build on the Juju controller which was bootstrapped and knows
+how to manage the SD-Core Control Plane Kubernetes cluster.
 
-First, create a Juju overlay file that specifies the Access and Mobility Management Function (AMF) host name and IP address for sharing with the radios. This host name must be resolvable by the gNB and the IP address must be reachable and resolve to the AMF unit. In the bootstrap step, we set the Control Plane MetalLB range to start at `10.201.0.52`, so that is what we use in the configuration.
+First, we will create a new Terraform module which we will use to deploy SD-Core Control Plane.
+In this module we will also specify the Access and Mobility Management Function (AMF)
+host name and IP address for sharing with the radios. This host name must be resolvable
+by the gNB and the IP address must be reachable and resolve to the AMF unit. In the bootstrap step,
+we set the Control Plane MetalLB range to start at `10.201.0.52`, so that is what we use
+in the configuration.
+Lastly, the module will expose the Software as a Service offer for the AMF.
 
-Log into the `juju-controller` VM.
+Log into the `juju-controller` VM:
 
 ```console
 multipass shell juju-controller
 ```
 
-Create the Juju overlay file where we specify the IP address and hostname for the AMF.
+Create Juju model for the SD-Core Control Plane:
+
+```console
+juju add-model control-plane control-plane-cluster
```

Comment:
Is this because tf cannot add a model on a specific cloud?

Reply:
Yes. It has some issues with credentials handling.